resources/assets/components/Signup/index.js

```diff
@@ -217,6 +217,18 @@ class Signup extends React.Component {
     this.hideHistory();
   }
 
+  // Set newstate with updated quantity and why_participated.
+  updateQuantityOrWhyState(quantity, why) {
```

Comment:
it is possible for this to just be `updateSignup(signup)` that just takes in a whole signup object and updates the state with that?

Reply:
oooo I like that idea! Just updated that!